src/instructlab/config/init.py

```diff
@@ -56,19 +56,20 @@
 @click.option("--train-profile", type=click.Path(), default=None)
 def init(
     interactive,
-    model_path,
+    model_path: str,
```

Comment:
the type in the @click.option is click.Path

Reply:
`type=click.Path` creates a function parameter of type `str` by default. The type annotation `str` is correct.

Reply:
Thank you for the clarification @tiran !

Reply:
++ the more you know!